### ISSUE TITLE
Maintenance status 

### ISSUE THREAD

**Issue by aminya** [ejfitzgerald/clang-tidy-cache#28]
Hi, I am reaching out since this project has not been maintained. I made some pull requests years ago, but they were left unanswered. Would you be open to adding maintainers to this repository or transferring the project to a new prospective maintainer like me?

https://github.com/ejfitzgerald/clang-tidy-cache/pull/10#issuecomment-1024840054

**Comment by kysucix** [comment]
I agree and support the proposal of aminya, it's a pity such a useful project is not maintained anymore.

**Comment by mathstuf** [comment]
@kokulshan has made an active fork and accepted my PRs (I've cross-filed them here for completion sake). I'm fine with working around that fork (luckily Golang does make migrating to forks easy as there's no central registry to deal with "ownership" over).

https://github.com/kokulshan/clang-tidy-cache
